distribution/images/multi-db/docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+services:
+  database:
+    image: ghcr.io/dyrector-io/dyrectorio/multi-database:1.0.0
+    container_name: database
+    restart: unless-stopped
+    healthcheck:
+      test: [CMD-SHELL, pg_isready -d crux -U crux_user]
+      interval: 5s
+      timeout: 5s
+      retries: 5
+    labels:
+      - org.dyrectorio.service-category=_internal
+    environment:
+      - POSTGRES_USER=${POSTGRES_ROOT_USER}
+      - POSTGRES_PASSWORD=${POSTGRES_ROOT_PASSWORD}
+      - CRUX_POSTGRES=${CRUX_POSTGRES}
+      - CRUX_POSTGRES_USER=${CRUX_POSTGRES_USER}
+      - CRUX_POSTGRES_PASSWORD=${CRUX_POSTGRES_PASSWORD}
+      - KRATOS_POSTGRES=${KRATOS_POSTGRES}
+      - KRATOS_POSTGRES_USER=${KRATOS_POSTGRES_USER}
+      - KRATOS_POSTGRES_PASSWORD=${KRATOS_POSTGRES_PASSWORD}
+    ports:
+      - "5432:5432"
+    volumes:
+      - database_data:/var/lib/postgresql/data
+      - ./init-database.sh:/docker-entrypoint-initdb.d/init-database.sh
```

Comment:
I see the init-db is mounted here, but not copied into the image nor put into the main `docker-compose.yaml`.
How should we use this? What's the idea?

Reply:
Ohh, I'll remove this. The `init-database.sh` file is needed for the Dockerfile.